### PR TITLE
Neurologists Form

### PR DESCRIPTION
**Pull request by davist11**
Also some minor nav adjustments to accommodate new link (and login button)

localhost_8000_neurologists

### CODE REVIEW

File: src/components/NeurologistsForm.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            .then(() => navigate(form.getAttribute('action')))
+            .catch((error) => alert(error))
+    }
+
+    const handleChange = (e) => {
+        setValues({ ...values, [e.target.name]: e.target.value })
+    }
+
+    return (
+        <form
```
Comment (davist11) [inline src/components/NeurologistsForm.js:45]: I was trying to make a `<Form />` component so I wouldn't have to duplicate but was struggling to figure out how to pass around the `handleChange` method to `{children}`

File: src/components/Layout.js [hunk 2]
```diff
@@ -47,6 +47,11 @@ const TemplateWrapper = ({ children }) => {
                     property="og:image"
                     content={`${withPrefix('/')}img/og-image.jpg`}
                 />
+                <link
+                    rel="stylesheet"
+                    type="text/css"
+                    href="https://cloud.typography.com/7033698/6288612/css/fonts.css"
+                />
```
Comment (jeremyfrank) [inline src/components/Layout.js:54]: Just a note to look into a few things later:

* Last time I used HF&J fonts from cloud.typography.com I learned that they base64 encode them into the css file, which is causes a huge amount of render blocking. You'll want to do some perf/render testing with and without this stylesheet, compare the differences, and determine if you need to make any adjustments.
* I did a quick search and landed on this support page about cloud.typography.com fonts not working on Netlify. Didn't look too closely at it though. https://community.netlify.com/t/headers-issue-with-cloud-typography-causing-403-error/642

File: src/components/NeurologistsForm.js (hunk 1 above)
Comment (jeremyfrank) [reply src/components/NeurologistsForm.js:45]: Was your intent to have `ContactForm` and `NeurologistForm` both load a generic `Form` component? If so, I think I like what you have here better, just the first two specific form components returning a `<form>` tag. If your methods above (encode, handleChange, handleSubmit) are what you're worried about duplicating, you could always move those to a ../utilities directory and import them at the top of each form component.

File: src/components/NeurologistsForm.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                )}
+                type="submit"
+            >
+                <span className={cx('flex justify-between items-center')}>
+                    <span>Submit</span>
+                    <span>
+                        <ArrowRight />
+                    </span>
+                </span>
+            </button>
```
Comment (jeremyfrank) [inline src/components/NeurologistsForm.js:128]: You could even make this a little `SubmitButton` component to share between both forms.

File: src/components/form/Textarea.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import Label from './Label'
+
+const Textarea = ({ label, name, handleChange }) => {
+    return (
+        <div>
```
Comment (leobauza) [inline src/components/form/Textarea.js:8]: i would bet eventually u'll need `className={className}` here for spacing and what not

Comment (davist11) [reply src/components/form/Textarea.js:8]: The `div` on `Field` has the spacing

File: src/components/NeurologistsForm.js (hunk 1 above)
Comment (davist11) [reply src/components/NeurologistsForm.js:45]: Exactly, but what you are saying makes sense, thanks.

File: src/components/form/Textarea.js (hunk 4 above)
Comment (leobauza) [reply src/components/form/Textarea.js:8]: oh if this div does nothing then use a fragment:

```
<>
  // your stuff here
</>
```

Comment (davist11) [reply src/components/form/Textarea.js:8]: 🤯 

had no idea, thanks

File: src/components/Layout.js (hunk 2 above)
Comment (davist11) [reply src/components/Layout.js:54]: I'll make sure to look into the performance stuff as we get into QA, but for now, it's loading pretty quickly. I had to add `<meta name="referrer" content="origin" />` to the `src/html.js` (wouldn't work in helmet) and that fixed font loading.

**Pull request merged**